src/olympia/users/models.py

```diff
@@ -176,6 +178,18 @@ def has_perm(self, perm, obj=None):
     def has_module_perms(self, app_label):
         return self.is_superuser
 
+    def has_read_developer_agreement(self):
+        if self.read_dev_agreement is None:
+            return False
+        if waffle.switch_is_active('post-review'):
+            # We want to make sure developers read the latest version of the
+            # agreement. The cutover date is the date the switch was last
+            # modified to turn it on. (When removing the waffle, change this
+            # for a static date).
+            switch = Switch.objects.get(name='post-review')
```

Comment:
I agree it's a bit weird... But the alternative was hardcoding a date that I didn't know was right :/